rosdep/python.yaml

```diff
@@ -6637,6 +6637,19 @@ python3-ezdxf:
     focal:
       pip:
         packages: [ezdxf]
+python3-fabric-pip:
```

Comment:
`python3-fabric` is available for Ubuntu Jammy. Whenever there is a system package available, that package should be used instead of pip. Older platforms which will never get the package as a system package can use pip.

Arch: https://archlinux.org/packages/community/any/fabric/
Debian: https://packages.debian.org/search?keywords=python3-fabric&searchon=names&suite=stable&section=all
Fedora: https://packages.fedoraproject.org/pkgs/python-fabric/python3-fabric/
nixOS: https://search.nixos.org/packages?channel=22.05&show=python310Packages.Fabric&from=0&size=50&sort=relevance&type=packages&query=python+Fabric
openSUSE: https://software.opensuse.org/package/python-Fabric
Ubuntu: https://packages.ubuntu.com/search?keywords=python3-fabric&searchon=names&suite=jammy&section=all

Please update this PR to add the system packages under the key `python3-fabric`, and add pip rules under that for any platforms which don't have system packages that you wish to support.